build.sbt

```diff
@@ -3,17 +3,19 @@ import sbt.Keys._
 lazy val pureCSV = crossProject.crossType(CrossType.Full).in(new File(".")).
   settings(
     name := "purecsv",
-    version := "0.0.7",
+    version := "0.0.8",
     organization := "com.github.melrief",
     publishMavenStyle := true,
     publishArtifact in Test := false,
-    scalaVersion := "2.11.7",
+    scalaVersion := "2.12.1",
+    crossScalaVersions := Seq("2.11.8", "2.12.1"),
+    javacOptions ++= Seq("-source", "1.8", "-target", "1.8"),
     scalacOptions ++= Seq("-feature", "-deprecation"),
     libraryDependencies ++= Seq(
-      "com.chuusai" %% "shapeless" % "2.2.5",
-      compilerPlugin("org.scalamacros" % "paradise" % "2.0.1" cross CrossVersion.full),
-      "org.scalatest" %% "scalatest" % "3.0.0-M7" % "test",
-      "com.github.marklister" %%% "product-collections" % "1.4.2"
+      "com.chuusai" % "shapeless_2.12" % "2.3.2",
```

Comment:
I think the `_2.12` here is making the compilation for `2.11` failing. We probably want to use `%% shapeless` here instead of `% shapeless_2.12`